### ISSUE TITLE
[macOS] there is terrible performance in fullscreen mode with the latest AIR version (476)

### ISSUE THREAD

**Issue by Oldes** [airsdk/Adobe-Runtime-Support#862]
## Problem Description

I made updated today one of our games using the latest AIR and in full-screen it's running with around 10fps instead of 60fps as it was before!
 
## Known Workarounds

Use older AIR... unfortunately I deleted older versions recently and don't know where to download them again.. I just found some really old AIR v256, which is OK.

@ajwfrost could you provide some link with older SDKs? Thanks.

**Comment by ajwfrost** [comment]
@Oldes  do you mean that the performance changed between one of our 33.1 releases and another ..? I don't recall anything changing on the macOS side recently, other than adding in the arm64 support for M1 platforms. Any chance we could try out your game?

thanks

**Comment by Oldes** [comment]
@ajwfrost Yes.. performance changed. I tried all SDKs backward and the last one which is fine is v345
I can send you link to archive with files which we use to build the game by email if you want.

**Comment by johnou** [comment]
@Oldes just to clarify, you are seeing the same performance regression with AIR 33.1.1.444 and 33.1.1.406?

**Comment by Oldes** [comment]
@johnou yes... I see the jerkiness since version 406.. And as I spend more time on it, I must say, that in version 345 the game is not running in full screen with 60fps, but rather at stable 30fps, while after version the frame times are very unstable:

Here is how it looks with v345:
![image](https://user-images.githubusercontent.com/454805/118837061-c00b5100-b8c4-11eb-8217-9d800c2ddf0f.png)

And this is from v476:
![image](https://user-images.githubusercontent.com/454805/118837278-f1841c80-b8c4-11eb-954b-2bd5652a17da.png)
(since the green spike it becomes stable as I turned off the full screen mode sooner)

**Comment by johnou** [comment]
@Oldes are you using the new M1 arm processor? or do you see the same issue on Intel procs too?

**Comment by chrslgr** [comment]
hi. we are seeing a performance loss when updating air from 33.1.1.217 to .633, .713, .929 (BigSur 11.6.3, i7). could you give us a hint how to measure performance like formerly in adobe scout to find the reason for that? (we are using intellij).
thanks, chris
